debezium-core/src/main/java/io/debezium/converters/CloudEventsConverterConfig.java

```diff
@@ -32,6 +33,10 @@ public class CloudEventsConverterConfig extends ConverterConfig {
             + "'avro' replaces the characters that cannot be used in the Avro type name with underscore (default)"
             + "'none' does not apply any adjustment";
 
+    public static final String CLOUDEVENTS_SOURCE_LOCATION_CONFIG = "source.location";
```

Comment:
IMHO this is misleading property name. You are setting location not only for `source` info block but for other metadata as well, so maybe it should be something like `metadata.location`?

Reply:
done